ipmininet/install/install.py

```diff
@@ -170,6 +171,27 @@ def install_openr(output_dir: str, may_fail=False):
         print("WARNING: Ignoring failed OpenR installation.", file=sys.stderr)
 
 
+def install_exabgp(output_dir: str, may_fail=False):
+    git_url = "https://github.com/Exa-Networks/exabgp.git"
+    exabgp_src_folder = "exabgp-%s-src" % ExaBGPVersion
+    exabgp_path_src_dir = os.path.join(output_dir, exabgp_src_folder)
+    exabgp_self_executable = os.path.join(output_dir, "exabgp")
+    final_link = "/usr/sbin/exabgp"
+
+    sh("git clone {url} -o {src_dir}".format(url=git_url, src_dir=exabgp_src_folder),
```

Comment:
```suggestion
    sh("git clone {url} {src_dir}".format(url=git_url, src_dir=exabgp_src_folder)
```

If you want the next command to work, you need to set the clone directory and not the origin with "-o" :)

Reply:
Yes indeed, it is more than likely that it was a bad copy and paste on my part. Anyway, it is changed a30819b8627b6c014fbe5949e7cf782dbbb8c413